### ISSUE TITLE
Local server doesn't start

### ISSUE THREAD

**Issue by ftyers** [jonorthwash/ud-annotatrix#317]
The error report should tell the user what to do.

```
$ npm run dev-server 

> ud-annotatrix@1.0.0 dev-server /home/fran/source/ud-annotatrix
> ANNOTATRIX_ENV=development nodemon server/app.js --ignore corpora

[nodemon] 1.18.3
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node server/app.js`
module.js:538
    throw err;
    ^

Error: Cannot find module 'moment'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/fran/source/ud-annotatrix/server/list-treebanks.js:5:16)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by jonorthwash** [comment]
I get this:
```
$ npm run dev-server

> ud-annotatrix@1.0.0 dev-server /data/apertium/git/UD/ud-annotatrix
> ANNOTATRIX_ENV=development nodemon server/app.js --ignore corpora

[nodemon] 1.18.3
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node server/app.js`
ConfigError: Please provide ANNOTATRIX_GH_CLIENT_SECRET
Express server listening at http://localhost:5316
{ port: 5316,
  host: 'localhost',
  protocol: 'http',
  corpora_path: 'corpora',
  secret: 'dev secret',
  environment: 'development',
  github: null,
  users_db_path: '.users',
  users: UsersDB { path: '.users.db' } }
```

**Comment by keggsmurph21** [comment]
fixed in 3a9a64b..4d4f9c6

```bash
$ npm run server

> ud-annotatrix@1.0.0 server /home/kmurphy4/ud-annotatrix
> node server/app.js || ./scripts/fail-nicely.sh

module.js:549
    throw err;
    ^

Error: Cannot find module 'mkdirp'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/kmurphy4/ud-annotatrix/server/config.js:3:16)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)


Fatal server error:

There is probably some helpful output above. Try running "npm install"
  before trying again.

For more info, check the README or file an issue at
  https://github.com/jonorthwash/ud-annotatrix/issues
```
